app/main/views/g_cloud.py

```diff
@@ -51,6 +51,21 @@
     You have too many results. Choose a category or add filters to refine your search.""")
 
 
+# TODO: Temporary redirect for direct_award internal IDs. Remove after external IDs have been out for a week - SW.
+@direct_award.before_request
+def redirect_internal_ids():
+    if 'project_id' in request.view_args:
+        project_id = request.view_args['project_id']
+
+        project = data_api_client.get_direct_award_project(project_id=project_id)['project']
```

Comment:
So if someone puts an internal ID in the url and we query the API for it, would we not get a 404 and this function would do nothing? Maybe I've misunderstood (in which case a slightly expanded description on the function might help). A unit test could also be a good addition to this, even though I know this is a temporary thing.

Could we po

Reply:
If someone puts an internal ID, they won't get a 404 because the API currently allows internal IDs to be used to retrieve resources.

Reply:
Ah yes, thanks